content/chronograf/v1.8/administration/managing-security.md

```diff
@@ -33,21 +35,23 @@ To configure any of the supported OAuth 2.0 providers to work with Chronograf,
 you must configure the `TOKEN_SECRET` environment variable (or command line option).
 Chronograf will use this secret to generate the JWT Signature for all access tokens.
 
-1. Generate a secret, high-entropy pseudo-random string.
+1. Generate a high-entropy pseudo-random string.
 
-    > For example, to do this with OpenSSL, run this command:
-    > ```
-    > openssl rand -base64 256 | tr -d '\n'
-    > ```
+   For example, to do this with OpenSSL, run this command:
+   ```sh
+   openssl rand -base64 256 | tr -d '\n'
+   ```
```

Comment:
indent lines 40-43
(line 45 currently renders as number 1)

Reply:
@pierwill, when I run locally, I still see line 45 as #1 

![image](https://user-images.githubusercontent.com/47797004/97624627-99e39980-19e4-11eb-90b6-bc9af80778be.png)

Reply:
To ensure content is properly nested in a list item (and not break the list numbering), indent the nested content with 4 space, not 2.